src/public/components/pretty-stringify.js

```diff
@@ -0,0 +1,7 @@
+import K, * as U from "karet.util"
+import React     from "karet"
+
+export const PrettyStringify = ( { space, obj } ) =>
+  <pre className="pretty-stringify">
+    { U.lift( x => JSON.stringify( x, null, Number( space ) ) )( obj ) }
+  </pre>
```

Comment:
I usually try to avoid having too much intelligence in program components (in general, not just UI components) and treat cases like that as bugs.  IOW, it would be responsibility of the user to make sure that `space` parameter is valid.